### PR TITLE
Uprade gradle wrapper, tools and dependencies

### PR DESCRIPTION
**Pull request by vRallev**

### CODE REVIEW

**Comment by StefMa** [comment]
I'm not the travis expert, but it seems you have to updated the `.travis.yml` as well? 🤔

**Comment by vRallev** [comment]
Yes, good catch.

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.2.2'
```
Comment (passsy) [inline build.gradle]: `2.2.3` is the latest version

Comment (vRallev) [inline build.gradle]: Done

File: build.gradle [hunk 2]
```diff
@@ -24,13 +24,19 @@ allprojects {
...
     MIN_SDK_VERSION = 14
-    TARGET_SDK_VERSION = 24
-    COMPILE_SDK_VERSION = 24
-    BUILD_TOOLS_VERSION = '24.0.3'
+    TARGET_SDK_VERSION = 25
+    COMPILE_SDK_VERSION = 25
+    BUILD_TOOLS_VERSION = '25.0.2'
 
+    supportLibraryVersion = '25.1.0'
+    compositeAndroidVersion = '25.0.1'
```
Comment (vRallev) [inline build.gradle]: I used this only for the time being. I reverted it now.

**Pull request merged**